Automattic/newspack-theme
# fix: correct button visited color

### All Submissions:


### Changes proposed in this Pull Request:

This PR fixes a specificity change in https://github.com/Automattic/newspack-theme/pull/2109 -- normally the text colour of the button would be overwritten in the colours PHP, but with it moving to CSS variables, the `:visited` colour of the button block ends up always being white, often incorrectly. 

See 1200550061930446-as-1206624531643238

### How to test the changes in this Pull Request:

1. Add a set of linked button blocks to a post or page -- include: 
* The default colours
* Custom colours using the palette (greys, or the primary, primary variation, or secondary variation colours)
* Fully custom colours using the colour pickers. 
2. Navigate to Customizer > Colors, and change the Secondary colour to a light colour. This is used by the button block by default, and will change the button text to black.
3. Click on your links and come back to the page; note that the visited link of the default block's text turns white: 

![image](https://github.com/Automattic/newspack-theme/assets/177561/e30bb041-8221-4eba-9d9d-431feb452e80)

4. Apply the PR and run `npm run build`.
5. Confirm that the default block's text stays black. 

![image](https://github.com/Automattic/newspack-theme/assets/177561/538ce8a4-89f8-4571-9f44-14a8fec20f2c)

6. Confirm the contrast on the other buttons still matches what you picked.
7. Try switching the Secondary colour back to a darker colour, and confirm that the visited button still looks good.

### Other information:





---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206624531643238
